docs/api-and-integration/octopus.migrator.exe-command-line/import.md

```diff
@@ -10,24 +10,6 @@ The export must have been made from an Octopus Server running the same release v
 Usage:
 
 ```text
-Octopus.Migrator import [<options>]
 
-Where `[<options>]` is any of:
-
-**Import options**
-
-Import:
-      --instance=VALUE       [Optional] Name of the instance to use
-      --directory=VALUE      Directory for imported files
-      --password=VALUE       Password to decrypt any sensitive values
-      --dry-run		     Do not commit changes, just print what would have happened
-      --overwrite            If a document with the same name already exists, it will be skipped by default.
-                             Use --overwrite to force it to be replaced.
-      --force 		     Imports even if there are validation errors
-                             (CAUTION: this may put the database in a bad state)
-      --include-tasklogs     Include the task log folder as part of the import process
- 
-Common Options:
-      --console              Don't attempt to run as a service, even if the user is non-interactive
-      --nologo               Don't print title or version information
 ```
+
```

Comment:
It's still not populating the import help :(

Reply:
Yeah - it should after `2019.5.8` goes out the door.